.golangci.yml

```diff
@@ -38,10 +38,21 @@ linters:
 issues:
```

Comment:
Some additional ideas for linters:
* exportloopref: checks for pointers to enclosing loop variables (https://github.com/kyoh86/exportloopref)
* gci: Gci control golang package import order and make it always deterministic. (https://github.com/daixiang0/gci)
  * would help to enforce imports in 3 groups: sdk, external, local (i.e. `goimports --local`)
* gofumpt: Gofumpt checks whether code was gofumpt-ed. (https://github.com/mvdan/gofumpt)
  * slightly more opinionated than gofmt (over time I found it pretty much matches my own preferences :))
* ifshort: Checks that your code uses short syntax for if-statements whenever possible (https://github.com/esimonov/ifshort)
* nilerr: Finds the code that returns nil even if it checks that the error is not nil. (https://github.com/gostaticanalysis/nilerr)

ref: https://golangci-lint.run/usage/linters/

Just some ideas. I don't want to start a lengthy discussion like tab vs spaces :). But if we get consensus on one / a few of them, that would be nice.

 If we want them, they should probably be added commented out here and then introduced one at a time (like with the `exclude` below)

Reply:
+1 on exportloopref, ifshort, and nilerr

The others require new tooling that's more stylistic than not, which I'm generally fine to leave as is